[IMP] General: Customer portal documentation update (#15438)
odoo/documentation: content/applications/general/users/portal.rst

:guilabel:`Action` drop-down menu located at the top-center of the interface.

.. image:: portal/grant-portal-access.png
   :alt: Use the Contacts application to give portal access to users.

Then select :guilabel:`Grant portal access`. A pop-up window appears, listing three fields:

- :guilabel:`Contact`: the recorded name of the contact in the Odoo database
- :guilabel:`Email`: the contact's email address that they will use to log into the portal
- :guilabel:`Latest Authentication`: the last time the user accessed the portal

> To grant portal access, first enter the contact's :guilabel:`Email` they will use to log into the

Reviewer: ```suggestion
To grant portal access, first enter the contact's :guilabel:`Email` used to log into the
```
